metrics-operator/converter/slo_converter.md

```diff
@@ -0,0 +1,303 @@
+# SLO Converter
+
+## Description
+
+SLO converter is a tool to convert `slo.yaml` files used in [KeptnV1](https://v1.keptn.sh/) into the new
+`AnalysisDefinition` resources used in the new kubernetes-native [Keptn](https://lifecycle.keptn.sh/).
+The converter is part of `metrics-operator` image.
+
+## Usage
+
+The converter will convert a single `slo.yaml` file into single `AnalysisDefintion` resource.
+
+To run the converter, execute the following command:
+
+```shell
+docker-run <METRICS_OPERATOR_IMAGE> manager --convert-slo=<PATH_TO_SLO> --slo-namespace=<ANALYSIS_VALUE_TEMPLATE_NAMESPACE> --definition=<DEFINITION_NAME>
+```
+
+Please be aware, you need to substitute the placeholders with the following information:
+
+* **PATH_TO_SLO** - path to your `slo.yaml` file
+* **ANALYSIS_VALUE_TEMPLATE_NAMESPACE** - namespace of `AnalysisValueTemplate` which will be referenced in objectives
+* **DEFINITION_NAME** - name of created `AnalysisDefinition`
+
+> **Note**
+
+All the SLOs present in `slo.yaml` file will reference `AnalysisValueTemplate` resources from the namespace defined
+by `ANALYSIS_VALUE_TEMPLATE_NAMESPACE` argument.
+
+## Conversion details
+
+We have multiple use-cases which are and which are not supported.
+There is a need to convert the use-cases that make
+logical sense and are common, but in some cases, where it is problematic and these cases will not be supported.
+
+> **Note** Please be aware, that comparison criteria containing `%` symbol ware not supported and will be ignored.
+
+### Unsupported use-cases
+
+Criteria with 3 and more inputs won't be supported, only the first 2 non-percentage inputs (those not containing `%`
+as we do not support comparison rules) will be taken and converted.
+In the example below, only `<600` and `>400` rules will be converted.
+Rule `>800` will be ignored.
+
+```yaml
+objectives:
```

Comment:
let's also add examples for how the resulting AnalysisDefinition for SLOs containing unsupported criteria would look like

Reply:
edit: I just saw that there is a example of a complete SLO -> AnalysisDefinition is available below. But I think it might also help to see the examples for the individual examples here to make it easier to see how such objectives end up looking like after the conversion